pkg/kn/commands/service/create_test.go

```diff
@@ -544,6 +544,43 @@ func TestServiceCreateMaxMinScale(t *testing.T) {
 	}
 }
 
+func TestServiceCreateScale(t *testing.T) {
+	action, created, _, err := fakeServiceCreate([]string{
+		"service", "create", "foo", "--image", "gcr.io/foo/bar:baz",
+		"--scale", "5",
+		"--concurrency-target", "10", "--concurrency-limit", "100",
```

Comment:
Why are these options important here ? I would just focus on `--scale` to make this test more focussed and easier to maintain (e.g. when something around concurrency-target would change in the future)

Reply:
As mentioned by @maximilien you should also add a test for the error case, i.e. when `--scale` is negative.

Reply:
And also a test for what happens when `--scale` and `--min-scale` & `--max-scale` are all provided (the more specific options should take precedence or an error should be thrown if both set of options are used)